packages/ndla-ui/src/Grid/Grid.tsx

```diff
@@ -6,102 +6,81 @@
  *
  */
 
-import { HTMLAttributes, ReactNode } from "react";
-import styled from "@emotion/styled";
-import { breakpoints, colors, misc, mq, spacing } from "@ndla/core";
+import { ReactNode } from "react";
+import { styled } from "@ndla/styled-system/jsx";
+
+const GridContainer = styled("div", {
+  base: {
+    display: "grid",
+    justifyContent: "center",
+    borderRadius: "xsmall",
+    padding: "xxlarge",
+    gridRowGap: "large",
+    gridColumnGap: "medium",
+    width: "100%",
+    marginBottom: "medium",
+    backgroundColor: "background.subtle",
+    maxWidth: "surface.4xlarge",
+    minWidth: "surface.xxsmall",
+    gridTemplateColumns: "repeat(2, minmax(0, 1fr))",
+    tabletDown: {
+      gridTemplateColumns: "repeat(1, minmax(0, 1fr))",
+      "& > div:nth-child(3):last-child": {
+        gridColumn: "span 1",
+      },
+    },
+    desktopDown: {
+      gridTemplateColumns: "repeat(2, minmax(0, 1fr))",
+      "& > div:nth-child(3):last-child": {
+        display: "flex",
+        flexFlow: "column",
+        justifyContent: "center",
+        alignItems: "center",
+        gridColumn: "span 2",
+      },
```

Comment:
Finnes det en måte å gjøre noe mellom to breakpoints eller må jeg gjøre det sånn og ha `gridColumn: "span 1"` i `tabletDown`?

Reply:
Du kan bruke mobileToTablet